brave/browser-laptop
# Fixing bookmarks toolbar "hover after click" behavior on folders

When context menu is expanded for a bookmark folder, mousing over other folders will auto-expand them (closing your previous folder's context menu).
Matching behavior in Chrome, mousing over non-folders will hide the menu.

Fixes https://github.com/brave/browser-laptop/issues/1725

Also includes:
- Added constant to faviconUtils for default icon size (referenced from bookmarksToolbar component)
- Moved bookmarksToolbar component and faviconUtils to new directory structure

Auditors: @bridiver 

## Test Plan
1. Launch Brave and ensure you have bookmarks toolbar showing with at least 2 folders and 1 non-folder
2. Click one of the folders in bookmarks toolbar and confirm it opens
3. Mouse over the other folder and confirm it auto-expands the menu
4. Mouse over a non-folder (ex: a bookmark) and confirm it hides the menu
5. Mouse back over a folder and confirm it auto-expands it again
6. Click anywhere to dismiss the context menu
7. Drag a bookmark over a bookmark folder and confirm it auto-expands. For bonus points, you can try dragging to a folder inside a folder.
